src/OpenDB/src/lefout/lefout.cpp

```diff
@@ -100,12 +106,39 @@ void lefout::writeBoxes(void* boxes, const char* indent)
   }
 }
 
+void lefout::writeHeader(dbBlock* db_block)
+{
+  dbTech* tech = db_block->getDataBase()->getTech();
+
+  char left_bus_delimeter = 0;
+  char right_bus_delimeter = 0;
+  char hier_delimeter = db_block->getHierarchyDelimeter();
+
+  db_block->getBusDelimeters(left_bus_delimeter, right_bus_delimeter);
+
+  if (left_bus_delimeter == 0)
+    left_bus_delimeter = '[';
+
+  if (right_bus_delimeter == 0)
+    right_bus_delimeter = ']';
+
+  if (hier_delimeter == 0)
+    hier_delimeter = '|';
+
+  writeVersion(tech->getLefVersionStr());
+  writeNameCaseSensitive(tech->getNamesCaseSensitive());
+  writeBusBitChars(left_bus_delimeter, right_bus_delimeter);
+  writeDividerChar(hier_delimeter);
+}
```

Comment:
wouldn't it simplify the code management if the `writeHeader(dbLib*)` were merged or called a common shared function? they look nearly identical.
How come `writeHeader(dbLib*)` writes units when this does not?

Reply:
I originally combined them into a `writeHeader(dbObject*)`, but the static_casts and enum checking around the way dbBlock and dbLib access Tech and delimiter information was a bit over complex to me. In all the abstract lef examples I'd seen none of the them had the units header so I removed it.